ci/integrationtest-run.yaml

```diff
@@ -1,11 +1,14 @@
 apiVersion: tekton.dev/v1beta1
 kind: TaskRun
 metadata:
-  generateName: it-test-
+  generateName: it-test-aws-
 spec:
   serviceAccountName: build-bot
   taskRef:
     name: integrationtest-gardenlinux-task
+  params:
+  - name: iaas
+    value: aws # or gcp
```

Comment:
how about also running it on gcp ?

Reply:
The taskrun is only an example to verify that the task _can_ execute in tekton.
To integrate it properly we need a pipeline (there you can run one task for aws and one for gcp in parallel).
However the pipeline integration with tekton does not make too much sense currently, as we don't have the means to get e.g. the ami-id or the gcp image-name out. Only with this information we can finally include the tests in some meaningful pipeline.